src/components/Side.tsx

```diff
@@ -1,17 +1,72 @@
 import React from 'react';
+import { Column, Container, Row } from 'nav-frontend-grid';
+import Lenke from 'nav-frontend-lenker';
 import './Side.sass';
-import { Container } from 'nav-frontend-grid';
+import Bedriftsmeny from '@navikt/bedriftsmeny';
+import { Link, useHistory } from 'react-router-dom';
+import { History } from 'history';
+import SoknadTittel from './SoknadTittel';
+import SideIndentering from './SideIndentering';
+import { Organisasjon } from '@navikt/bedriftsmeny/lib/organisasjon';
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
+import { useArbeidsgiver } from '../context/ArbeidsgiverContext';
 
 interface SideProps {
   children: React.ReactNode;
   className?: string;
+  sidetittel: string;
+  title: string;
+  subtitle: string;
+  bedriftsmeny?: boolean;
 }
 
+const IngenTilgangAdvarsel = () => (
```

Comment:
Du kan til og med ha denne i en egen fil, om du synes denne blir for lang

Reply:
God ide